tensorflow_probability/examples/vae.py

```diff
@@ -79,7 +79,7 @@
     new_mu[p] := 0
     new_Sigma[p] := eye(d)
     new_mu[q] := inv(chol(Sigma[p])) @ (mu[p] - mu[q])
-    new_Sigma[q] := inv(Sigma[q]) @ Sigma[p]
+    new_Sigma[q] :=  I -  inv(Sigma[q])
```

Comment:
What's the rationale for this change? Doing a change of variables, one should expect the covariance to transform with the inverse covariance (rather than a complement).